crates/bitwarden-crypto/src/uniffi_support.rs

```diff
@@ -43,3 +59,11 @@ impl UniffiCustomTypeConverter for UnsignedSharedKey {
         obj.to_string()
     }
 }
+
+// Uniffi doesn't emit unused types, this is a dummy record to ensure that the custom type
+// converters are emitted
```

Comment:
Can be removed when https://github.com/bitwarden/sdk-internal/pull/221 is merged.